### ISSUE TITLE
[FEA] epsilon kwarg for LogOp and possible rename

### ISSUE THREAD

**Issue by benfred** [NVIDIA/NVTabular#24]
<a href="https://github.com/alecgunny"><img src="https://avatars0.githubusercontent.com/u/14932242?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [alecgunny](https://github.com/alecgunny)**
_Monday May 11, 2020 at 16:05 GMT_
_Originally opened as https://github.com/rapidsai/recsys/issues/105_

----

1. Add `eps` kwarg to `LogOp` such that the math implemented is `log(x+eps)` instead of only `log(x+1)`. I get the need for numeric stability but this might be confusing for some users who haven't read the source and expected log to just mean log, especially when they try to transform back to the original space (for e.g. metric calculation). Small values like `1e-7` could be useful to users who want "log-like" behavior but still need numeric stability.

2. `LogOp` is the only `Op` that has "Op" in its name. Might be worth just calling `Log` for consistency.

**Comment by benfred** [comment]
<a href="https://github.com/benfred"><img src="https://avatars2.githubusercontent.com/u/69536?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [benfred](https://github.com/benfred)**
_Monday May 11, 2020 at 16:13 GMT_

----

+1 to both of these

**Comment by rdipietro** [comment]
Additional motivation: Numerically stable implementations of certain losses require working in log space, and making use of standard operations like `torch.logsumexp` (for example). In the current form,

1. If a user is unaware of the name inconsistency, they'll get incorrect results.

2. Even if a user is aware of this inconsistency, forming numerically stable implementations becomes difficult: we first need to essentially invert the 'add 1 and take log op` before we proceed.
